src/React.re

```diff
@@ -28,18 +28,30 @@ external createElementVariadic:
   (component('props), 'props, array(element)) => element =
   "createElement";
 
-[@bs.module "react"] [@deprecated "Please use JSX syntax directly."]
+[@bs.module "react/jsx-runtime"]
 external jsxKeyed: (component('props), 'props, string) => element = "jsx";
 
-[@bs.module "react"] [@deprecated "Please use JSX syntax directly."]
+[@bs.module "react/jsx-runtime"]
 external jsx: (component('props), 'props) => element = "jsx";
 
-[@bs.module "react"] [@deprecated "Please use JSX syntax directly."]
+[@bs.module "react/jsx-runtime"]
 external jsxs: (component('props), 'props) => element = "jsxs";
 
-[@bs.module "react"] [@deprecated "Please use JSX syntax directly."]
+[@bs.module "react/jsx-runtime"]
 external jsxsKeyed: (component('props), 'props, string) => element = "jsxs";
 
+type fragmentProps('children) = {children: 'children};
+
+/* [@bs.module "react/jsx-runtime"] */
+/* external jsxFragment: component('props) = "Fragment"; */
+[@bs.module "react/jsx-runtime"] external jsxFragment: 'a = "Fragment";
+/* external jsxFragment: component({. "children": element}) = "Fragment"; */
+/* [@bs.obj] */
+/* external makeProps: */
+/* (~children: element, ~key: 'key=?, unit) => {. "children": element}; */
+[@bs.module "react"]
+external make: component({. "children": element}) = "Fragment";
```

Comment:
I'm missing something here: this is under React and doesn't seem right, would it make more sense under a `Fragment` module (not really important the name since this is called from inside the ppx

Reply:
Why is this not supposed to be React.Fragment? Just like React.Suspense this should live here, no?

Reply:
Sure, but it's currently not under Fragment module.

Reply:
oh I see what you're saying. I think this is an oversight, will look into it again.